src/features/content-answers/Answer.tsx

```diff
@@ -1,20 +1,26 @@
 import * as React from 'react';
 
 import ContentAnswersGridCard from './ContentAnswersGridCard';
+import InlineError from './InlineError';
 import LoadingElement from './LoadingElement';
+// @ts-ignore: no ts definition
+// eslint-disable-next-line import/named
+import { ElementsXhrError } from '../../../common/types/api';
 
 import './Answer.scss';
 
 type Props = {
     answer?: string;
+    error?: ElementsXhrError;
     isLoading: boolean;
 };
 
-const Answer = ({ answer, isLoading }: Props) => {
+const Answer = ({ answer, error, isLoading }: Props) => {
     return (
         <div className="bdl-Answer">
             {answer && <ContentAnswersGridCard>{answer}</ContentAnswersGridCard>}
             {!answer && isLoading && <LoadingElement />}
```

Comment:
We need to prevent these were rendering if there is an error, definitely the first line but the second line is debatable depending on how isLoading is handle but I think its safe to say both should not render if there is an error